pubspec.yaml

```diff
@@ -21,11 +21,11 @@ dependencies:
   yaml: ^3.1.0
 
 dev_dependencies:
-  build_runner: ^2.0.0
-  json_serializable: ^4.1.0
-  test: ^1.15.7
+  build_runner: ^2.1.2
```

Comment:
`dart pub upgrade --null-safety` raises minimums to the latest version on the same major that null-safety was introduced. I walked it back for the actual dependencies to avoid as many resolution issues as possible, but left it for the `dev_dependencies` because it doesn't affect consumers.

Reply:
that's fair